tests/service.netcore/NuGet.Config

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <!-- MSBuild -->
+    <add key="dotnet-buildtools" value="https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
```

Comment:
Copied this over directly from the nuget.config in the `service` project, no changes.  This will go away when I put the netcore fsproj back into the same folder as the old fsproj.